tests/app/vertx-opentracing/README.md

```diff
@@ -12,44 +12,63 @@ Manager is performed via publishing a message on a shared destination.
+This example features a pub-sub architecture via an event bus, using [Eclipse Vert.x](http://vertx.io). Each `Verticle` is a
+microservice and is deployed into a different service under OpenShift. Hawkular APM acts here as an OpenTracing provider and is
+able to capture data from all services, displaying this data on its UI, like the screenshots below:
 
-First step is to start up the APM server.
+![Distributed tracing - Aggregated view](screenshot-1.png?raw=true "Distributed tracing - Aggregated view")
 
-Then open a console window from this folder and build the module:
 
-```shell
-$ mvn clean install -Pitest
-```
-Note: the `itest` profile is required if intending to run using docker.
+![Distributed tracing - Instance view](screenshot-2.png?raw=true "Distributed tracing - Instance view")
 
-Now you can run the example vert.x application, using either maven:
+## Setting it up
 
-```shell
-$ export HAWKULAR_APM_URI=http://localhost:8080
-$ export HAWKULAR_APM_USERNAME=jdoe
-$ export HAWKULAR_APM_PASSWORD=password
-$ mvn exec:java
-```
-or docker:
+To run this, you'll need an OpenShift cluster (local or otherwise), with Hawkular APM running on it. This can be accomplished
+with the following commands on Fedora 24 (or later):
 
-```shell
-$ docker run -it --rm -p 8180:8180 -e HAWKULAR_APM_URI=http://172.17.0.1:8080 -e HAWKULAR_APM_USERNAME=jdoe -e HAWKULAR_APM_PASSWORD=password hawkular/apm-tests-app-vertx-opentracing
-```
+    sudo iptables -F
+    oc cluster up
+    oc create -f https://raw.githubusercontent.com/jboss-dockerfiles/hawkular-apm/master/openshift-templates/hawkular-apm-server-deployment.yml
 
-or OpenShift:
+Once Hawkular APM is up and running, each microservice needs to be added to OpenShift. Please, refer to the individual
+`README.md` files for instructions, but in general, it involves creating a new build, starting the build and creating a new app,
+setting the appropriate environment variables for the service to reach the Hawkular APM server. Services that should be exposed to the
+public will need also an `expose` command.
 
-```shell
-mvn clean install
-oc new-build --binary --name=vertx-opentracing -l app=vertx-opentracing
-oc start-build vertx-opentracing --from-dir=. --follow
-oc new-app vertx-opentracing HAWKULAR_APM_URI="http://hawkular-apm" HAWKULAR_APM_USERNAME="admin" HAWKULAR_APM_PASSWORD="password" -l app=vertx-opentracing
-oc expose service vertx-opentracing
-```
+One example of the steps might be:
+
+    oc new-build --binary --name=order-manager -l app=order-manager
+    oc start-build order-manager --from-dir=. --follow
+    oc new-app order-manager HAWKULAR_APM_URI="http://hawkular-apm" HAWKULAR_APM_USERNAME="admin" HAWKULAR_APM_PASSWORD="password"
+    oc expose service order-manager
+
+Once all microservices are set and ready, the following command can be issued to test:
+
+    curl -X POST -d '{"accountId":"fred","itemId":"laptop","quantity":2}' http://order-manager-myproject.192.168.178.26.xip.io/orders
+
```

Comment:
Maybe it would be good to add go to Hawkular APM UI `https://hawkular-apm-myproject.<oc cluser IP>.xip.io`